### PR TITLE
fix: issue of image not deleting and using X button

### PR DESCRIPTION
**Pull request by sudoDeznit**
fix: issue of image not deleting when deleting from  AddTags page using X button.

- the issue was image were not removed from asyncstorage

### CODE REVIEW

**Comment by sudoDeznit** [comment]
fix : issue of app crashing on delete from deleting from AddTags page.

- issue was swiperIndex not changed on image delete
- so in getTags() of AddTags.js it was jumping to else condition even when webPhotos were not there. so added a if condition to handle that and a final else condition to handle any other condition.

**Comment by sudoDeznit** [comment]
this issue solves #59 
app crashes  on deleting last image using X icon on add tags page this exists in v2.4 abdroid.
@xlcrr

File: screens/pages/components/LitterBottomSearch.js [hunk 1]
```diff
@@ -118,6 +120,9 @@ class LitterBottomSearch extends PureComponent {
                 {
                     text: 'OK',
                     onPress: () => {
+                        this.props.swiperIndexChanged(
+                            currentIndex > 0 ? currentIndex - 1 : 0
+                        );
```
Comment (xlcrr) [inline screens/pages/components/LitterBottomSearch.js:125]: Why do we need to change the index here?

Comment (sudoDeznit) [reply screens/pages/components/LitterBottomSearch.js:125]: To keep state in sync and remove any possibility of encountering "index out of bounds" error.

Comment (xlcrr) [reply screens/pages/components/LitterBottomSearch.js:125]: I see. Can you wrap the condition in brackets so it becomes `(currentIndex > 0)` ?

**Review by xlcrr — APPROVED** [review]

**Pull request merged**